src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts

```diff
@@ -293,11 +293,14 @@ export class HaVoiceCommandDialog extends LitElement {
         ha-button-menu a {
           text-decoration: none;
         }
-
         .pipelines-loading {
           display: flex;
           justify-content: center;
         }
+        ha-assist-chat {
```

Comment:
@wendevlin I am not happy with this solution as we shouldn't mess with styling inside components. I couldn't find anything else in the brief time I had allocated for this. Got an idea? (Feel free to push to this PR)

Reply:
Mhh, but the title of your PR says exactly this that you want to change styling outside. We could work with a variable to set default values.

Reply:
Exactly. Styling a custom elmeent on the outside (margin) is fine. Adding padding to an element means that the element could break this accidentally in the future. However in this case, I am unable to get it working with the required flex box/height without triggering a scroll wheel.